html/pfappserver/root/package.json

```diff
@@ -20,6 +20,7 @@
     "messageformat": "^2.3.0",
     "mime-types": "^2.1.30",
     "papaparse": "5.3.0",
+    "pinia": "^0.4.1",
```

Comment:
Can we already use release [0.5.2](https://github.com/posva/pinia/blob/v1/CHANGELOG.md)?